plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteClient.java

```diff
@@ -193,7 +193,20 @@ public IgniteClient(
     @Override
     public Collection<String> listSchemas(Connection connection)
     {
-        return ImmutableSet.of(IGNITE_SCHEMA);
+        try (ResultSet resultSet = connection.getMetaData().getSchemas()) {
```

Comment:
Could you leave a code comment why we need to override this method? Code readers will wonder because it's pretty similar to `BaseJdbcClient#listSchemas`

Reply:
Ok